CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+### 2.32.0
+This release includes:
+* Fluent Bit [1.9.10](https://github.com/fluent/fluent-bit/tree/v1.9.10)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.4
+* Amazon Kinesis Streams for Fluent Bit 1.10.2
+* Amazon Kinesis Firehose for Fluent Bit 1.7.2
+
+Compared to `2.31.12.20230911` this release adds:
+* Feature - Amazon Managed Prometheus support added via sigv4 authentication on `prometheus_remote_write` [aws-for-fluent-bit:256](https://github.com/aws/aws-for-fluent-bit/issues/256)
+* Enhancement - Allow multiple `cloudwatch_log` output workers by removing sequence tokens from API requests [aws-for-fluent-bit:526](https://github.com/aws/aws-for-fluent-bit/issues/526)
```

Comment:
Let's link to the new firelens example somewhere in the release notes.

Reply:
the final link is predictable

Reply:
Sure! Link added.